x-pack/plugins/enterprise_search/public/applications/workplace_search/components/layout/personal_dashboard_sidebar/private_sources_sidebar.tsx

```diff
@@ -31,10 +34,16 @@ export const PrivateSourcesSidebar = () => {
     ? PRIVATE_CAN_CREATE_PAGE_DESCRIPTION
     : PRIVATE_VIEW_ONLY_PAGE_DESCRIPTION;
 
+  const {
+    contentSource: { id = '', name = '' },
+  } = useValues(SourceLogic);
+
+  const navItems = [{ id, name, items: useSourceSubNav() }];
+
   return (
     <>
       <ViewContentHeader title={PAGE_TITLE} description={PAGE_DESCRIPTION} />
-      <SourceSubNav />
+      {id && <EuiSideNav items={navItems} mobileBreakpoints={[]} />}
```

Comment:
Whoops, forgot the most important comment here!

⚠️ **CI will fail because of this line!** ⚠️ That is totally expected.

I'm waiting for https://github.com/elastic/kibana/pull/101334 to land with EUI v34.x so I can get the `mobileBreakpoints` prop. Otherwise the current side nav collapses to mobile right now when it shouldn't :dead_inside:

Once the above PR lands (fingers crossed, should be soon - this is reviewable anyway without it) I can just merge upstream master into PR and it will start working.

Reply:
If the above PR doesn't land by Monday also I will likely comment out the prop or put a typescript ignore for now 🤔 (if you have a preference, lmk)

Reply:
Still 4 more approvals on the above EUI PR to go, and it looks like Security has a merge conflict... so I'm gonna go ahead and ignore the Typescript error on this for now and merge as-is. Could use some help remembering to uncomment the ignore after 7.14FF as I've got a lot of balls to juggle and will likely forget 🙇‍♂️

Reply:
https://github.com/elastic/kibana/pull/102592/commits/57dcd76e7f18d771bf08ab0b5d9b626761ad2cbe

Reply:
> Could use some help remembering to uncomment the ignore after 7.14FF 

https://github.com/elastic/workplace-search-team/issues/1796